acceptance/acceptance_test.go

```diff
@@ -223,6 +223,10 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 
 	if !isTruePtr(config.Cloud) && cloudEnv != "" {
 		t.Skipf("Disabled via Cloud setting in %s (CLOUD_ENV=%s)", configPath, cloudEnv)
+	} else {
+		if isTruePtr(config.RequiresUnityCatalog) && os.Getenv("TEST_METASTORE_ID") == "" {
```

Comment:
Do we need to add interaction between this option and Cloud? Cannot this be an independent condition?

Reply:
In local tests you won't have TEST_METASTORE_ID set up so the test would be skipped locally if we don't test for a cloud env as well. Any proposals on how this can be done differently?

Reply:
> In local tests you won't have TEST_METASTORE_ID set up so the test would be skipped locally if we don't test for a cloud env as well.

That's okay. We can add later TEST_METASTORE_ID to local tests if we need it.